### ISSUE TITLE
arlo cam settings error

### ISSUE THREAD

**Issue by max-at** [scryptedapp/arlo#35]
I have connection issues to my Arlo Pro 4 since 2 days... don't know why, has arlo changed anything on server-side?

at the camera settings page in scrypted it says "Settings Error", but i didn't change anything since months...

did an update to the newest beta 0.11.22, but also no luck sadly

here my latest log from the camera page:

[https://logpaste.com/L10DD7Cv](url)

**Comment by bjia56** [comment]
Can you provide the logs from the plugin page, instead of the camera page?

**Comment by max-at** [comment]
sure :-)

[https://logpaste.com/xiTNs6zP](url)

**Comment by max-at** [comment]
ok i now changed the 2fa from SMS to eMail, and than back to SMS, reloaded the Plugin, got a new SMS-Verification-Code then i entered the new code and saved it, works now again... tried to force the reauthentication earlier, but with no success, got some errors in the log, didn't think of changing the 2fa methods... but that was the problem indeed... thx for your help mate

**Comment by max-at** [comment]
good idea, but i didn't log out of a different session in the last days, maybe auto-disconnect on the server side?

**Comment by bjia56** [comment]
We should be all set on this one. If there's anything else, please open a new issue.
